internal/config/config.go

```diff
@@ -47,6 +55,8 @@ type Config struct {
 	InitContainerImages   []string          `desc:"List of init containers that should be appended for each deployment that has Config.Annotation" split_words:"true"`
 	ContainerImages       []string          `desc:"List of containers that should be appended for each deployment that has Config.Annotation" split_words:"true"`
 	Envs                  []string          `desc:"Additional Envs that should be appended for each Config.ContainerImages and Config.InitContainerImages" split_words:"true"`
+	WebhookMode           string            `default:"spire" desc:"Set to 'secret' to use custom certificates from k8s secret. Set to 'selfregister' to use the automatically generated webhook configuration" split_words:"true"`
```

Comment:
Could we consider how are enums parsing with envconf.
```suggestion
	WebhookMode           Mode            `default:"spire" desc:"Set to 'secret' to use custom certificates from k8s secret. Set to 'selfregister' to use the automatically generated webhook configuration" split_words:"true"`
```

Reply:
Thanks, fixed